pull_request_template.md

```diff
@@ -1,14 +1,14 @@
 **Related issues**
 
-*Add links to related issues here. Use [keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) if you want an issue to be automatically closed when the PR is merged.*
+<!-- *Add links to related issues here. Use [keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) if you want an issue to be automatically closed when the PR is merged.* -->
```

Comment:
We can remove the asterisks now, too.

Reply:
And put the URL in parentheses, instead of making a Markdown link.

Reply:
Done in https://github.com/open-contracting/infrastructure/pull/374/commits/4847b59fe2410a82822b834c51c0cd4b605022cb